apps/sms/views/inbox/js/inbox.js

```diff
@@ -163,8 +164,7 @@ var InboxView = {
     var threadId = node.dataset.threadId;
     var draftId = node.dataset.draftId;
 
-    var threadOrDraft = draftId ?
-      Drafts.byDraftId(+draftId) : Threads.get(+threadId);
+    var threadOrDraft = Threads.get(+(draftId || threadId));
```

Comment:
Event that second commit is just for the Demo purpose, I still think we'll need something very similar, how do you feel about ```thread.draft``` property?